src/deadline_worker_agent/log_messages.py

```diff
@@ -386,13 +391,14 @@ class SessionLogEventSubtype(str, Enum):
     COMPLETE = "Complete"
     INFO = "Info"  # Generic information about the session
     LOGS = "Logs"  # Info on where the logs are going
+    OPENJD = "OpenJD"  # Logs from the openjd.sessions module applicable to the worker log
 
 
 class SessionLogEvent(BaseLogEvent):
     ti = "🔷"
     type = "Session"
-    queue_id: str
-    job_id: str
+    queue_id: Optional[str]
+    job_id: Optional[str]
```

Comment:
The logging docs don't indicate that these are optional. Was this a leftover from a previous iteration? Looks to me like they can remain required.

Reply:
It's optional because of this line: https://github.com/aws-deadline/deadline-cloud-worker-agent/pull/422/files#diff-a0d5ca678872256e58410f3ff16c7792477e639aa7fc6f3b716b027ef243457aR635

It's not guaranteed that the session exists in the dictionary we're checking, in this case I biased to keep the log as a session log, just one that doesn't have queue or job info.

I can modify the docs to reflect this

Reply:
Added in the latest commit

Reply:
That line should never be false though, right? These log records only come from the openjd-sessions logger, and an openjd-sessions logger can only have things put into it for a specific session-id if that session-id has already been added to the session map, no? The session map contains the mapping of session-id to a worker_agent.sessions.Session, and we create one of those before we create the openjd.sessions.Session.

Reply:
It's more about when the session ID is removed from the session map, I've gone into a bit more detail below